src/app/containers/ConsentBanner/Banner.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+import {
+  ConsentBanner,
+  ConsentBannerButton,
+  ConsentBannerLink,
+  ConsentBannerText,
+} from '../../components/ConsentBanner';
+
+const Accept = (message, onClick) => (
+  <ConsentBannerButton onClick={onClick}>{message}</ConsentBannerButton>
+);
+
+const Reject = (message, href, onClick) => (
+  <ConsentBannerLink href={href} onClick={onClick}>
+    {message}
+  </ConsentBannerLink>
+);
+
+const Text = ({ first, linkText, linkUrl, last }) => {
+  const Link = linkUrl && linkText ? <a href={linkUrl}>{linkText}</a> : null;
+
+  return (
+    <ConsentBannerText>
+      {first}
+      {Link}
+      {last}
+    </ConsentBannerText>
+  );
+};
+
+Text.propTypes = {
+  first: string.isRequired,
+  linkText: string.isRequired,
+  linkUrl: string.isRequired,
+  last: string.isRequired,
+};
```

Comment:
Maybe move lines 21-38 out to a `BannerText.jsx` file (and rename to `BannerText`.

Reply:
done with a folder :)